ftp_app_main.py

```diff
@@ -0,0 +1,22 @@
+from ftp_app_view import ftp_app_view
+import queue
+import threading
+import time
+
+def main():
+    for i in range(5):
+        time.sleep(1)
+        print(i)
+
+if __name__ == "__main__":
+
+    app = ftp_app_view(queue.Queue(), queue.Queue())
```

Comment:
Probably want to extract these queues into variables and pass them in as args so that you can pass the queues to your other threads to use. You'll need at least 2 threads communicating over these queues for them to be useful. For example:
```python
status_queue = queue.Queue()
job_queue = queue.Queue()

app = ftp_app_view(status_queue, job_queue)
worker_thread = threading.Thread(target=work, args=(status_queue, job_queue))
```

Note: Have you looked into whether the `queue.Queue` implementation is thread-safe?